### PR TITLE
Pad around publication subheadings

### LINKED ISSUE #2165: Fix the styling on the publication page
The [one example](https://portal.hubmapconsortium.org/publication/kidney-cell-states-atlas) publication isn't linked yet from the home page. (The TMC needs to configure it.)

With the #2164, Nils confirmed that the structure is right, but the rendering isn't pretty. Might just be a matter of getting the right styling on section titles, but loop in Tiffany if needed.

### PR DESCRIPTION
**Pull request by mccalluc**
Fix #2165?  @tsliaw , does this look ok?
Screen Shot 2021-11-29 at 9 40 30 AM

### CODE REVIEW

File: context/app/static/js/pages/Publication/style.js [hunk 1]
```diff
@@ -4,6 +4,10 @@ import Paper from '@material-ui/core/Paper';
 const StyledPaper = styled(Paper)`
   padding: 20px 40px 20px 40px;
   margin-bottom: 16px;
+
+  h2 {
+    padding: 20px 0 20px 0;
+  }
```
Comment (john-conroy) [inline context/app/static/js/pages/Publication/style.js:10]: Sorry if jumping in here is wrong, but I prefer to style the component itself unless it's somehow inaccessible.

Comment (mccalluc) [reply context/app/static/js/pages/Publication/style.js:10]: The h2 here is raw html that we generate from the markdown and inject into react... so I think this is a case where it is inaccessible... I could add a comment to that effect?

(Jumping in is welcome! I might be misunderstanding something.)

Comment (john-conroy) [reply context/app/static/js/pages/Publication/style.js:10]: You could use shorthand `padding: 20px 0px;`, but not a big deal.

**Review by john-conroy — APPROVED** [review]

File: context/app/static/js/pages/Publication/style.js [hunk 2]
```diff
@@ -4,6 +4,10 @@ import Paper from '@material-ui/core/Paper';
 const StyledPaper = styled(Paper)`
   padding: 20px 40px 20px 40px;
   margin-bottom: 16px;
+
+  h2 {
+    padding: 20px 0 20px 0;
```
Comment (mccalluc) [inline context/app/static/js/pages/Publication/style.js:9]:
```suggestion
    padding: 20px 0px;
```

**Pull request merged**